lib/generators/shopify_app/install/templates/embedded_app.html.erb

```diff
@@ -1,24 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <title>Shopify Embedded Example App</title>
+  <head>
+    <meta charset="utf-8" />
+    <title>Shopify Embedded Example App</title>
+    <%= stylesheet_link_tag 'application' %>
+    <%= csrf_meta_tags %>
+  </head>
 
-  <script src="//cdn.shopify.com/s/assets/external/app.js?<%= Time.now.strftime('%Y%m%d%H') %>"></script>
-  <script type="text/javascript">
-    ShopifyApp.init({
-      apiKey: "<%= ShopifyApp.configuration.api_key %>",
-      shopOrigin: "<%= "https://#{ @shop_session.url }" if @shop_session %>",
-      debug: <%= Rails.env.development? ? 'true' : 'false' %>
-    });
-  </script>
+  <body>
+    <div class="app-wrapper">
+      <div class="app-content">
```

Comment:
I’m going to do this anyway because it’s not really harmful, it'll be useful to us internally, and 3rd parties without the gem can still use it if they want. :)